src/pages/Homepage/Homepage.vue

```diff
@@ -112,55 +107,32 @@ export default {
 		},
 		result({ data }) {
 			// Explicit lender preferences experiment - EXP-GROW-166-Aug2020
```

Comment:
Confirmed with Brandon that we can remove this experiment, it is 100% shown at this time.